README.md

```diff
@@ -1,3 +1,3 @@
 # Internal Survey Tool
 
-To handle all the surveys within an organization.
+To handle and manage all the employee surveys within an organization.
```

Comment:
Can you remove `the`?

`To handle and manage all employee surveys within an organization.`

Reply:
Sure got it!